src/controllers/playback/video/index.js

```diff
@@ -1219,6 +1219,12 @@ export default function (view) {
             case 'Enter':
                 showOsd();
                 break;
+            case 'PageUp':
+                playbackManager.nextChapter(currentPlayer);
+                break;
+            case 'PageDown':
+                playbackManager.previousChapter(currentPlayer);
+                break;
```

Comment:
I'm curious. Why did they climb so high?

Reply:
No functional reason, other than it was easier on the eyes to organize by which needed the preventDefault and which didn't